tycho-p2-repository-plugin/src/main/java/org/eclipse/tycho/plugins/p2/repository/MavenP2SiteMojo.java

```diff
@@ -485,7 +486,7 @@ protected File createMavenAdvice(Artifact artifact) throws MojoExecutionExceptio
             addProvidesAndProperty(properties, TychoConstants.PROP_CLASSIFIER, artifact.getClassifier(), cnt++);
             addProvidesAndProperty(properties, "maven-scope", artifact.getScope(), cnt++);
             try (OutputStream os = new BufferedOutputStream(new FileOutputStream(p2))) {
-                properties.store(os, null);
+                ReproducibleUtils.storeProperties(properties, os, null);
```

Comment:
As it seems always the same boiler-plate, maybe we can have a utils method that accepts a file ans creates the stream and alike 

`ReproducibleUtils.storeProperties(properties, p2);`

Reply:
Yes, why not.

Reply:
Too bad that this is private: https://github.com/apache/maven-archiver/blob/ffca18fc302c9f19cf2e5bfe7aad4a89a5d83fc6/src/main/java/org/apache/maven/shared/archiver/PomPropertiesUtil.java#L51